features/tests/test_models.py

```diff
@@ -44,8 +46,16 @@ def test_feature():
     assert Feature.objects.count() == 1
 
 
+def test_one_liner_length_limit():
+    too_long = "a" * 51
+
+    with pytest.raises(DataError):
```

Comment:
I was sort of thinking that you could add this field e.g. to the GQL query on `test_features_query` which would then also test that we are returning a value for this field in the API.

Reply:
This particular I think is testing that Django internals (which are tested by Django itself) are working as expected. I would say that we might be safe without specifically testing that `max_length` is in effect.

If we want to test that the limit on that field is 50 I would move this test to `test_models` and do it in a slightly different way:
```python
from django.apps import apps
ft = apps.get_model("features", "FeatureTranslation")
field = ft._meta.get_field("one_liner")
assert field.max_length == 50
```

Reply:
it is in the test_models file, api test added.

Reply:
as to how to test it, I like your way as well as being more explicit, but my point was that we are testing that it's impossible to save an object when one of the fields has certain property, not testing what value the property has in the code. I guess the difference is just in subtle difference in perspectives =)

Reply:
I would probably still just remove this test since we do not have any tests for other fields that have defined the `max_length` and testing that `max_value` behaves as we expect (i.e. throws an error) is something that is tested in the Django library itself i.e. testing the same here is redundant.

Reply:
that it is possible to save value under 64 is being implicitly tested when a feature is created with non-empty field. =) I can remove the test, sure, if it does not go in line with other tests in the app.